packages/foreman/rubygem-deep_cloneable/rubygem-deep_cloneable.spec

```diff
@@ -70,6 +70,9 @@ cp -pa .%{gem_dir}/* \
 %doc %{gem_instdir}/readme.md
```

Comment:
Could you add `%doc %{gem_instdir}/CHANGELOG.md` here?

Reply:
sure, done